mk/artifact.mk

```diff
@@ -0,0 +1,68 @@
+ENABLE_PREBUILT ?= 1
+
+CC ?= gcc
+CROSS_COMPILE ?= riscv-none-elf-
+
+BINDIR := $(abspath $(OUT))
+
+TEST_SUITES += \
+	ansibench \
+	rv8-bench
+
+# "ieee754" needs F extension
+# "smolnes", "ticks" have inline assembly and only work in riscv
+TESTBENCHES += \
+	captcha \
+	donut \
+	fcalc \
+	hamilton \
+	jit \
+	lena \
+	line \
+	maj2random \
+	mandelbrot \
+	nqueens \
+	nyancat \
+	pi \
+	puzzle \
+	qrcode \
+	richards \
+	rvsim \
+	spirograph \
+	uaes
+
+SHELL_HACK := $(shell mkdir -p $(BINDIR)/linux-x86-softfp $(BINDIR)/riscv32)
+
+ifeq ($(call has, PREBUILT), 1)
```

Comment:
`ENABLE_PREBUILT` should be set explicitly in advance.

Reply:
It has been set at first line and is enabled in default.